src/createEpicMiddleware.ts

```diff
@@ -30,7 +31,7 @@ export function createEpicMiddleware<
   // other RxJS code outside of redux-observable internals.
   const QueueScheduler: any = queueScheduler.constructor;
   const uniqueQueueScheduler: typeof queueScheduler = new QueueScheduler(
-    (queueScheduler as any).SchedulerAction
+    (queueScheduler as any).SchedulerAction || (queueScheduler as any).schedulerActionCtor
```

Comment:
Should this just be `(queueScheduler as any).schedulerActionCtor`? I don't think this release would be backward compatible, so there may be no need to keep `ScheduleAction`

Reply:
Since we now require Rxjs 7, it's fine to break compatibility with 6.